### PR TITLE
ci: replace actions-rs/cargo with naive command

### PR DESCRIPTION
**Pull request by skbeh**
This avoids node 12 warnings.

### CODE REVIEW

**Review by tiann — APPROVED** [review]

**Comment by tiann** [comment]
This doesn't trigger CI?

File: .github/workflows/ksud.yml [hunk 1]
```diff
@@ -29,11 +29,7 @@ jobs:
         restore-keys: ${{ runner.os }}-cargo-
 
     - name: Build ksud
-      uses: actions-rs/cargo@v1
-      with:
-        use-cross: true
-        command: build
-        args: --target ${{ inputs.target }} --release --manifest-path ./userspace/ksud/Cargo.toml
+      run: cross build --target ${{ inputs.target }} --release --manifest-path ./userspace/ksud/Cargo.toml
```
Comment (tiann) [inline .github/workflows/ksud.yml:32]: when the `cross` is installed?

Comment (Ylarod) [reply .github/workflows/ksud.yml:32]: seems built-in in action's ubuntu image

Comment (skbeh) [reply .github/workflows/ksud.yml:32]: According to https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md, it is included in the action image.

**Pull request merged**